src/components/form/form_row/form_row.tsx

```diff
@@ -220,18 +220,21 @@ export class EuiFormRow extends Component<EuiFormRowProps, EuiFormRowState> {
 
     let optionalLabel;
     const isLegend = label && labelType === 'legend' ? true : false;
+    const labelId = `${id}_label`;
```

Comment:
This should be kebab-case and not snake_case to match all our other IDs in this component `{id}-row`, `{id}-help`, and `{id}-error`

```suggestion
    const labelId = `${id}-label`;
```